app/presenters/finder_presenter.rb

```diff
@@ -149,6 +149,10 @@ def filter_sentence_fragments
     filters.map(&:sentence_fragment).compact
   end
 
+  def show_keyword_search?
+    keywords.present? || facets.any? || results.total.positive?
+  end
```

Comment:
Yes it looks like this will cover all the scenarios where the results could be filtered by the user.

I suppose this could also be used for showing a different response message, as it might be confusing to have no search input without an explanation as to why.

Reply:
Yeah, we'll have a think about that 🤔